src/util/std_expr.cpp

```diff
@@ -43,6 +43,58 @@ exprt conjunction(const exprt::operandst &op)
   }
 }
 
+// Create a lexicographic less-than relation between two tuples of variables.
+// This is used in the implementation of multidimensional decreases clauses.
+exprt create_lexicographic_less_than(
+  const std::vector<symbol_exprt> &small,
+  const std::vector<symbol_exprt> &large)
+{
```

Comment:
I'd call the parameters `lhs` and `rhs`, to be consistent with the other binary relations.